### DISCUSSION TITLE
I am unable to create the Anvil network in metamask

### DISCUSSION THREAD

**Discussion by Harikrishnan2004** [Cyfrin/foundry-full-course-cu#3130] [General]
I am unable to create the Anvil network in metamask, how to solve this issue, i can't click the save button 
![Screenshot 2024-12-30 182658](https://github.com/user-attachments/assets/d370be0c-d3f5-4585-8179-24bf05f45c73)

**Comment by EngrPips** [comment]
Hello @Harikrishnan2004, use `http://localhost:8545` instead of `localhost:8545` and lets see if that helps.

**Reply by Harikrishnan2004** [reply]
Hi @EngrPips, thanks for the reply but I found the issue, first I need to run the Anvil server in VS code and then I should have filled that form, that was the issue and now its working fine, thanks for responding one again.

**Reply by EngrPips** [reply]
Anytime, Keep crushing it.
